src/main/java/se/sundsvall/precheck/integration/AbstractIntegrationProperties.java

```diff
@@ -0,0 +1,33 @@
+package se.sundsvall.precheck.integration;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Duration;
+
+@Getter
+@Setter
+public abstract class AbstractIntegrationProperties {
+
+    private String baseUrl;
+    private Duration readTimeout = Duration.ofSeconds(300);
+    private Duration connectTimeout = Duration.ofSeconds(30);
+
+    @Getter
+    @Setter
+    public static final class BasicAuth {
```

Comment:
Why is this needed in this service?

Reply:
It was a class that came with the 'AbstractIntegrationProperties', and you are correct that it has no use in this service. 

It has now been removed.